_includes/search.html

```diff
@@ -8,9 +8,21 @@
   role="search"
 >
   <input name="utf8" type="hidden" value="&#x2713;" />
-  <input type="hidden" name="affiliate" id="affiliate" value="developers-login-gov" autocomplete="off" />
+  <input
```

Comment:
Didn't mean to update this, but I'm guessing Prettier auto-formatting took effect here on save. Seems safe enough to keep and probably good to normalize on Prettier format anyways.